lib/uploadcare/client/webhook_client.rb

```diff
@@ -9,8 +9,12 @@ module Client
     class WebhookClient < RestClient
       # Create webhook
       # @see https://uploadcare.com/docs/api_reference/rest/webhooks/#subscribe
-      def create(target_url, event: 'file.uploaded', is_active: true)
-        body = { 'target_url': target_url, 'event': event, 'is_active': is_active }.to_json
+      def create(options = {})
+        body = {
+          'target_url': options[:target_url],
+          'event': options[:event] || 'file.uploaded',
+          'is_active': options[:is_active] || true
```

Comment:
It seems that we get `"is_active": true` if we send `is_active: false`. Is this OK?

Reply:
Maybe use
```ruby
'is_active': options[:is_active].nil? ? true : options[:is_active]
```

Reply:
And possibly we should add a test for this case.